src/gps/GPS.h

```diff
@@ -154,27 +156,14 @@ class GPS : private concurrency::OSThread
      * calls sleep/wake
      */
     void setAwake(bool on);
-    void doGPSpowersave(bool on);
     virtual bool factoryReset();
 
     // Creates an instance of the GPS class.
     // Returns the new instance or null if the GPS is not present.
     static GPS *createGps();
 
   protected:
-    /// If possible force the GPS into sleep/low power mode
-    virtual void sleep();
-
-    /// wake the GPS into normal operation mode
-    virtual void wake();
-
     /** Subclasses should look for serial rx characters here and feed it to their GPS parser
```

Comment:
comment is not terminated... got compiler warnings

src/gps/GPS.h:168:5: warning: "/*" within comment [-Wcomment]